lib/internal/per_context/primordials.js

```diff
@@ -409,9 +409,6 @@ const SafePromise = makeSafe(
   }
 );
 
-primordials.PromisePrototypeCatch = (thisPromise, onRejected) =>
-  PromisePrototypeThen(thisPromise, undefined, onRejected);
-
```

Comment:
Where was that decision made?

I think it's absurd to claim the users of primordials have any expectations whatsoever about their implementation, only about what they do (especially since ALL of them are call-bound, which means NONE of them have their original value). If the lint rule would be preventing them from using PromisePrototypeCatch, how is that aligned with their expectations, but "PromisePrototypeCatch is made safe" isn't?

Reply:
The linter rule is easy to workaround (you can add a `// eslint-disable` comment), while changing the primordials value makes it impossible to access the original `%Promise.prototype.catch%` reliably.

Reply:
Since it's call-bound, you can never access it anyways. When would someone need a bind-alias to the original?

Reply:
Hmm, I think the disable comment should be used only when there is a good reason to override a good rule, not whenever a bad rule kicks in.

I gotta second Jordan: I would find it very unintuitive to use `PromisePrototypeThen()` instead of `PromisePrototypeCatch()`. I might use `PromisePrototypeThen()`, when I have both success AND failure callbacks, but never just for failure, unless I was forced to (and if I was forced to, I'd be silently cursing whoever—and I like you, so I don't wanna do that 😆).

Reply:
I would honestly remove all "raw" promise calls for async/await since that always produces a safe native promise as far as I'm aware.

Reply:
I agree with Benjamin, if that rule can encourage contributors to use async/await rather than chaining promises, I'm calling it a win. As I said above, we can always add a `SafePromisePrototypeCatch` later if necessary, and in the meantime, I'm OK with being silently cursed :)